views/patterns/02-organisms/global/footer.blade.php

```diff
@@ -67,7 +67,6 @@
   </div> <!-- /.c-footer--inner -->
 </footer> <!-- /.c-footer -->
 
-<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
 <script>
   $jq2 = $.noConflict(true);
```

Comment:
Does this line need to be there? My understanding is that is just to tell jQuery to look at the `2.2.4` version, not the WP shipped version.